rust/agents/relayer/src/msg/metadata/multisig/base.rs

```diff
@@ -89,6 +89,28 @@ pub trait MultisigIsmMetadataBuilder: AsRef<MessageMetadataBuilder> + Send + Syn
         let metas: Result<Vec<Vec<u8>>> = self.token_layout().iter().map(build_token).collect();
         Ok(metas?.into_iter().flatten().collect())
     }
+
+    async fn ism_validator_requirements(
+        &self,
+        ism_address: H256,
+        message: &HyperlaneMessage,
+    ) -> Result<(Vec<(H256, u64)>, u64)> {
```

Comment:
would it make more sense to represent the validators and weights as a mapping offchain?

Reply:
I don't understand like validator[weight] = 13e9?